packages/core/src/Services/HttpServer/HttpServer.ts

```diff
@@ -67,6 +76,7 @@ export class HttpServer {
       hostname: host,
       port,
       fetch: this.handleRequest.bind(this),
+      plugins: hasWebsocketsEnabled ? [this.gWebsocketService.serverPlugin] : []
```

Comment:
We need to find a solution to define plugins for `srvx` globally - not just `wss`, since more things might be injected later.
As before, you can’t inject `wss` directly here.

A possible solution could be creating a core internal service like:

```ts
export class PluginsService {
  private fServerPlugins = [];

  public registerPlugin(...) {
    // logic to store plugin
  }
}
```

Then the `core` package can export this service.
Inside the WebSocket service, you would register the plugin through this service instead of injecting `wss` directly.